rust/src/adapter/datadog.rs

```diff
@@ -275,6 +320,35 @@ impl DatadogAdapter {
             self.trace_events.clear();
         }
 
+        // if metric_events.len() > 0 {
+        //     let socket = UdpSocket::bind("0.0.0.0:9999").unwrap();
+        //     for stat in metric_events {
+        //         // TODO we are just assumign they are statsd format but we can check stat.format
+        //         let message = format!("{}|#trace_id:{}", stat.message, stat.trace_id.unwrap());
+        //         // TODO get the host from config
+        //         socket
+        //             .send_to(message.as_bytes(), "127.0.0.1:8125")
+        //             .unwrap();
+        //     }
+        // }
+
+        // TODO find a better way to get logs to the agent
+
+        // if log_events.len() > 0 {
+        //     let file = OpenOptions::new()
+        //         .write(true)
+        //         .append(true)
+        //         .create(true)
+        //         .open("/tmp/planktonic.log")?;
+
+        //     let mut file_writer = io::BufWriter::new(file);
+        //     for evt in log_events {
+        //         dbg!(&evt);
+        //         file_writer.write_all(evt.message.as_bytes()).unwrap();
+        //     }
+        //     file_writer.flush()?;
+        // }
+
```

Comment:
Leaving all this stuff disabled for now. I'll start a new PR with some of the implementation details so we can demo. I didn't want this to break someone in the mean time.